indexer/services/comlink/public/api-documentation.md

```diff
@@ -3094,6 +3097,7 @@ or
       "side": "BUY",
       "size": "string",
       "price": "string",
+      "type": "LIMIT",
```

Comment:
are you manually editing this? It should be autogenerated when `comlink` is built

Reply:
ah yeah, cc @johnqh